inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/AuditController.java

```diff
@@ -59,4 +62,17 @@ public Response<Boolean> refreshCache() {
         return Response.success(auditService.refreshBaseItemCache());
     }
 
+    @ApiOperation(value = "insert a source and connect to the source")
+    @PostMapping(value = "/audit/updateSource")
+    public Response<Boolean> updateAuditQuerySource(@RequestBody UpdateAuditSourceRequest request) {
+        return Response.success(
+                auditService.updateAuditQuerySource(request));
+    }
+
+    @ApiOperation(value = "query which source is connected")
+    @GetMapping("/audit/querySource")
```

Comment:
1. Please change the API and method name to `getSource`, in order to facilitate the subsequent expansion of the listSources interface, the interface format will be unified.

2. Entity is used by DB and cannot be returned directly as a parameter. You should learn from other interfaces, such as getComponentHeartbeat, to convert Entity into Response and return it.